src/pages/home/report/ContextMenu/ReportActionContextMenu.js

```diff
@@ -29,11 +30,12 @@ function showContextMenu(
     onHide = () => {},
     isArchivedRoom = false,
     isChronosReport = false,
+    childReportID = '',
```

Comment:
```suggestion
    childReportID = '0',
```

shouldn't the default value here be '0'?